oss-cloud-frontend/bucket-policy.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "PublicReadGetObject",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "s3:GetObject",
+      "Resource": "arn:aws:s3:::oss-cloud-frontend-dev/*"
+    }
+  ]
+}

```

Comment:
How is this file used?

Reply:
Frontend is deployed to S3 bucket, which follows the policy defined in this file. We pushed the policy from this file and decided not to delete it, in order to have it saved in repository,